app/scripts/components/common/card.tsx

```diff
@@ -284,6 +285,38 @@ const CardFigure = styled(Figure)`
   }
 `;
 
+const ExternalLinkMark = styled.div`
+  display: flex;
+  align-items: center;
+  position: absolute;
+  top: ${variableGlsp(0.25)};
+  right: ${variableGlsp(0.25)};
+  padding: ${variableGlsp(0.125)} ${variableGlsp(0.25)};
+  background-color: ${themeVal('color.primary')};
+  color: ${themeVal('color.surface')};
+  text-transform: none;
+  border-radius: calc(
+    ${multiply(themeVal('shape.rounded'), 2)} - ${variableGlsp(0.125)}
```

Comment:
So @faustoperez shared this cool trick on how to derive the value for nested border-radius, which should be (outside border-radius - gap ): https://www.30secondsofcode.org/css/s/nested-border-radius/ But if we take it as it is, the value becomes 0, so I halved the value for the gap here. :[